tests/features/plugins.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { mount, config } from '../../src'
+import { WrapperAPI } from '../../src/types'
+
+declare module '../../src/vue-wrapper' {
+  // @ts-ignore
+  interface VueWrapper {
```

Comment:
`@ts-ignore` should be removed here: you should just need to have the same signature than `VueWrapper`, i.e `VueWrapper<T extends ComponentPublicInstance>` and TS will be happy to merge the declarations.
You should also be able to remove all the other `@ts-ignore` in the file.

Reply:
🤔 I will give this a try

Reply:
To no-one's surprised I changed the types based on @cexbrayat 's recommendation and it worked